x/btcstaking/types/genesis.go

```diff
@@ -28,6 +31,16 @@ func (gs GenesisState) Validate() error {
 		if len(receiver.Addr) == 0 {
 			return fmt.Errorf("receiver addr cannot be empty")
 		}
+		if len(receiver.EthAddr) == 42 {
+			if receiver.EthAddr[:2] != "0x" {
+				return fmt.Errorf("receiver's eth addr must start with 0x")
+			}
+			if _, err := hex.DecodeString(receiver.EthAddr[2:]); err != nil {
```

Comment:
Can be changed to `common.FromHex(receiver.EthAddr)`

Reply:
done